src/foam/nanos/actioncommand/ActionCommand.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Wrong copyright header. Use the smaller non-Google one for code that NP contributes.